### PR TITLE
Adding Time to Leave project

### PR DESCRIPTION
**Pull request by thamara**
Adding Time to Leave project.
Time to Leave repository: https://github.com/thamara/time-to-leave

### CODE REVIEW

File: projects/thamara/time-to-leave.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - Developers
+  - Backend Developers
+  - Frontend Developers
+  - Designers
+  - Testers
+contributionOverview:
+  mainLocation: Brazil
+  isMentorshipAvailable: false
+featuredMap:
+  url: https://app.codesee.io/maps/public/798d8e40-19c3-11ec-b873-35f335ec68d2
```
Comment (deammer) [inline projects/thamara/time-to-leave.mdx:24]: **praise:** this map looks absolutely fantastic! I've shared it with my colleagues at CodeSee and we're all extremely impressed 💯

**Review by deammer — APPROVED** [review]
@thamara thank you for adding your project to OSS port! I've streamlined your list of tags a little bit, but otherwise, your page looks great. It will go live in a few minutes. Cheers!

File: projects/thamara/time-to-leave.mdx (hunk 1 above)
Comment (thamara) [reply projects/thamara/time-to-leave.mdx:24]: Oh! Thank you!
It's a really cool tool!

Comment (LizardEatsFlies) [reply projects/thamara/time-to-leave.mdx:24]: its true, its a really neat map! i love the colors you chose and its super clear. Very nice

**Comment by jleven** [comment]
@thamara -- Thank you again for listing your project, and for listing your CodeSee Map on our [showcase page](https://app.codesee.io/maps/community). I'm thrilled to tell you the team absolutely loves your map, and we've selected it to be one of our first "Featured Projects" on that page!

And thank you again for being part of the OSS Port community.

**Pull request merged**